sceptre/diffing/stack_differ.py

```diff
@@ -44,13 +84,22 @@ class StackDiffer(Generic[DiffType]):
     As an abstract base class, the two comparison methods need to be implemented so that the
     StackDiff can be generated.
     """
-
     STACK_STATUSES_INDICATING_NOT_DEPLOYED = [
         'CREATE_FAILED',
         'ROLLBACK_COMPLETE',
         'DELETE_COMPLETE',
     ]
 
+    NO_ECHO_REPLACEMENT = "***HIDDEN***"
```

Comment:
Since CloudFormation always returns NoEcho parameters as "****", this will always show up as a difference. We simply cannot know if the parameter value has changed from production for NoEcho parameters _anyway_, no matter what we do. So it's simply a matter of how we want to render local values that Sceptre would send for NoEcho parameters. By default, we will mask them with this.